### PR TITLE
added clear error messages for different failures sending remote logs

### PR DESCRIPTION
**Pull request by david-granulating**
Resolve: https://app.clickup.com/t/86byqmh6t

### CODE REVIEW

**Comment by Jongy** [comment]
I'll review it next week.

File: glogger/sender.py [hunk 1]
```diff
@@ -20,14 +20,19 @@
...
 from requests.auth import HTTPBasicAuth
 
 from glogger.messages_buffer import MessagesBuffer
 
 from .stdout_logger import get_stdout_logger
 
-SERVER_SEND_ERROR_MESSAGE = "Error posting logs to server"
+SENDER_CONNECTION_ERROR_MESSAGE = "REMOTE_LOGGER: Failed establishing connection to logs server, check log server url"
+SENDER_TIMEOUT_MESSAGE = "REMOTE_LOGGER: Timeout occurred while sending logs to server"
+SENDER_UNAUTHORIZED_MESSAGE = "REMOTE_LOGGER: Authentication error while sending logs to server, check gprofiler token"
```
Comment (Jongy) [inline glogger/sender.py:33]: Authentication or authorization? These are different, use one 😅

File: glogger/sender.py [hunk 2]
```diff
@@ -157,8 +162,22 @@ def send(self) -> None:
...
+        except requests.exceptions.ConnectionError:
+            self.stdout_logger.error(SENDER_CONNECTION_ERROR_MESSAGE)
+        except requests.exceptions.Timeout:
+            self.stdout_logger.error(SENDER_TIMEOUT_MESSAGE)
+        except requests.exceptions.HTTPError as err:
+            if err.response.status_code == 401:
+                self.stdout_logger.error(SENDER_UNAUTHORIZED_MESSAGE)
+            elif err.response.status_code == 500:
+                self.stdout_logger.error(
+                    f"REMOTE_LOGGER: Received 500 from server, gprofiler token is probably invalid / missing. "
```
Comment (Jongy) [inline glogger/sender.py:174]:
```suggestion
                    f"REMOTE_LOGGER: Received 500 from server, token is probably invalid / missing. "
```

**Review by Jongy — CHANGES_REQUESTED** [review]
2 small comments

**Comment by david-granulating** [comment]
@Jongy fixed 2 small comments

**Review by Jongy — APPROVED** [review]

**Pull request merged**